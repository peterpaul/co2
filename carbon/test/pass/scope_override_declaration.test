["stdio.h"] int printf(char * format, ...);

int first = 1;

class Test {
	int second = 1;
	void test(int third) {
		printf ("first: %d\n", first);
		printf ("second: %d\n", second);
		printf ("third: %d\n", third);
		int first = 2, second = 2, third = 2;
		printf ("first: %d\n", first);
		printf ("second: %d\n", second);
		printf ("third: %d\n", third);
	}
}

int main() {
	Test test = new Test();
	test.test(1);
	return 0;
}
