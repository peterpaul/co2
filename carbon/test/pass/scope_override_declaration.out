first: 1
second: 0
third: 1
first: 2
second: 2
third: 2
